docs/sources/dashboards/create-reports/_index.md

```diff
@@ -0,0 +1,331 @@
+Refer to specific guides to understand what permissions are required.
+{{< /admonition >}}
+
+Any changes you make to a dashboard used in a report are reflected the next time the report is sent. For example, if you change the time range in the dashboard, then the time range in the report also changes, unless you've configured a custom time range.
+
+## Requirements
+
+- SMTP must be configured for reports to be sent. Refer to [SMTP](ref:smtp) in [Configuration](ref:configuration) for more information.
+- The Image Renderer plugin must be installed or the remote rendering service must be set up. Refer to [Image rendering](ref:image-rendering) for more information.
+
+## Access control
+
+When [RBAC](ref:rbac) is enabled, you need to have the relevant [Permissions](ref:permission) to create and manage reports.
+
+## Create a report
+
+Only organization administrators can create reports by default. You can customize who can create reports with [Role-based access control](ref:role-based-access-control).
+
+The report creation process is multi-step, but you don't need to complete these steps in order and you can skip steps by clicking a step name at the top of the page:
+
+![Reporting wizard](/media/docs/grafana/dashboards/screenshot-reporting-wizard-v11.5.png)
+
+To create a report, follow these steps:
+
+1. Click **Dashboards > Reports** in the main menu.
+1. Click **+ Create a new report**.
+1. Complete the report steps, as needed; you don't need to complete these steps in order and you can skip steps by clicking a step name at the top of the page:
+   - [Select dashboard](#1-select-dashboard)
+   - [Format report](#2-format-report)
+   - [Schedule](#3-schedule)
+   - [Share](#4-share)
+   - [Confirm](#5-confirm)
+1. Click one of the following buttons in the top-right corner of the screen:
+   - **Send** or **Schedule send** - The report is sent according the schedule you've set.
+   - **Save as draft** - You can save a draft at any point during the report creation or update process, even if it's missing required fields. The report won't be sent according to its schedule while it's a draft.
+   - **Discard** - Delete the report draft. This action can't be reversed.
+
+### 1. Select dashboard
+
+At this step, select the dashboard or dashboards on which the report is based, as well as the variables and time range for those dashboards. The options are:
+
+<!-- prettier-ignore-start -->
+
+| Option | Description |
+| ------ | ----------- |
+| Source dashboard | Select the dashboard from which you want to generate the report. |
+| [Template variables](#template-variables) | Select the variable **Datasource** and **Instance** for the selected dashboard. This option is only displayed if the dashboard has variables. |
+| [Time range (optional)](#time-range) | By default, reports use the saved time range of the dashboard. Optionally, you can change the time range of the report. |
```

Comment:
I would mention here that in order to always use the dashboard time range, it needs to be blank by clicking the X at the end of the input since this is hard to find.

Reply:
See my comment above, this would be prefilled only if you create a report from a dashboard. Maybe we should change the `By default` to say `If empty` so it's always true.